packages/web-components/fast-components/src/accordion/accordion-item/accordion-item.ts

```diff
@@ -0,0 +1,25 @@
+import { attr, booleanConverter, FASTElement, observable } from "@microsoft/fast-element";
+
+export class AccordionItem extends FASTElement {
+    @attr({ attribute: "heading-level", mode: "fromView", converter: booleanConverter })
```

Comment:
Should this be using `booleanConverter`?

Reply:
heading-level likely should not

Reply:
since its defaulted, we only want heading level to display if the author assigns one.

Reply:
fromView should be good for this - we'd use both here only if this were a boolean value which needs converting

Reply:
I think I was confused on how the mode `fromView` works thinking it needed that converter, Removed.